source/contributing/guidelines.rst

```diff
@@ -35,8 +35,8 @@ General steps
     `Minor Issues on GitHub <https://github.com/search?q=org%3ASpongePowered+Ongoing+Minor+Issue+List&type=Issues>`_.
 
 4. If the issue requires a bigger change you may want to submit the issues without the necessary changes first, so we
-   can confirm the issue and know that you're working on fixing it. You should also create a WIP (work in process) pull
-   request prefixed with ``[WIP]`` early so we can already start reviewing them.
+   can confirm the issue and know that you're working on fixing it. You should also create a draft pull
+   request or comment with ``~wip`` so we can already start reviewing them.
```

Comment:
I am not certain of what is meant by
> or comment with ``~wip``

Should the comment be made in the issue or the pull request?

Perhaps one of the following two sentences would be clearer:

You should also create a draft pull request **with a** ``~wip`` **comment** so we ...

OR

You should also create a draft pull request or comment **in the issue** with ``~wip`` so we ...

Reply:
In theory WIP status can be automatically set in GitHub now. More detail is probably required.

Reply:
I see. I found the following URLs. Maybe a link in a ``Tip`` block to one of them for more information will suffice. 

https://github.blog/2019-02-14-introducing-draft-pull-requests/

https://help.github.com/en/articles/about-pull-requests#draft-pull-requests

Reply:
I was referring to limbo.
https://github.com/SpongePowered/limbo-config/wiki/Pull-Request-Commands